template/package.json

```diff
@@ -4,16 +4,23 @@
   "version": "1.0.0",
   "author": "{{ author }}",
   "private": true,
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
```

Comment:
Does this file get generated automatically? I couldn't see it

Reply:
yes, if you're building a npm-module.